tests/check/check_test.go

```diff
@@ -37,9 +37,10 @@ type stage struct {
 }
 
 type assertion struct {
-	Tuple       *pb.TupleKey
-	Expectation bool
-	Trace       string
+	Tuple            *pb.TupleKey
+	ContextualTuples []*pb.TupleKey `yaml:"contextualTuples"`
```

Comment:
What's the default behavior here? I thought `ContextTuples` would, by default, marshal to camelCase `contextualTuples`.

Reply:
If you do add this, please also add it to `oldcheck/check_test.go`. But yeah, I think the default is this behavior so you should be able to omit it 🤔 ?

Reply:
That's what you would think right? But it seems it is necessary.